TideApp/TideApp/Domain/Network/Requests/GetWeatherInformationRequest.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  GetWeatherInformationRequest.swift
+//  TideApp
+//
+//  Created by John Sanderson on 19/05/2021.
+//
+
+import CoreLocation
+import Combine
+
+struct GetWeatherInformationRequest: RequestType {
```

Comment:
The idea is that this "Request" can be made in multiple locations and it hides all the logic around fetching from the API and mapping to a `WeatherInfo` object etc.

Reply:
👏